content/tutorial-build-run-rippled-ubuntu.md

```diff
@@ -0,0 +1,184 @@
+# Build and Run `rippled` on Ubuntu
+
+`rippled` is the core peer-to-peer server that manages the XRP Ledger. A `rippled` server can connect to a network of peers, relay cryptographically signed transactions, and maintain a local copy of the complete shared global ledger.
+
+For an overview of `rippled`, see [Operating rippled Servers](tutorial-rippled-setup.html).
+
+Use these instructions to build a `rippled` 0.90.0+ binary file and run it as a stock `rippled` server on Ubuntu 15.04 or later. ***TODO: Question: I added the specific `rippled` release series to this text - is that okay? Should I remove the reference to Ubuntu 15.04 and just say that this is for Ubuntu 16.04 LTS or later?*** These instructions were tested on Ubuntu 16.04 LTS.
+
+For information about building `rippled` for other platforms, see [Builds](https://github.com/ripple/rippled/tree/develop/Builds) in the `rippled` GitHub repository.
+
+
+## System Requirements
+
+**_To build `rippled`:_**
+
+You need a **minimum** of 8GB of RAM.
+
+**_To run `rippled`:_**
+
+Meet these [system requirements](tutorial-rippled-setup.html#system-requirements).
+
+
+## 1. Build `rippled`
+
+These instructions use Ubuntu's APT (Advanced Packaging Tool) to install the software prerequisites you need to build and run `rippled`.
+
+1. Update the list of packages that are available for `apt-get` to install or upgrade.
+
+        sudo apt-get update
+
+2. Upgrade currently installed packages.
+
+        sudo apt-get -y upgrade
+
+3. Install Git.
+
+        sudo apt-get -y install git
+
+4. Install SCons.
+
+        sudo apt-get -y install scons
+
+5. Install `pkg-config`.
+
+        sudo apt-get -y install pkg-config
+
+6. Install Protocol Buffers.
+
+        sudo apt-get -y install protobuf-compiler
+        sudo apt-get -y install libprotobuf-dev
+
+7. Install Secure Socket Layer (SSL) toolkit development files.
+
+        sudo apt-get -y install libssl-dev
+
+8. Install `wget` to be able to download Boost in the next step.
+
+        sudo apt-get install wget
+
+8. Compile Boost.
+
+    Starting in `rippled` 0.90, the compatible Boost version is 1.65.0. Because Boost version 1.65.0 isn't available in the Ubuntu 16.04 repos, you'll need to compile it yourself. ***TODO: Per Scott D, the recommendation will be either 1.65.0 or 1.66.0+ for 0.90.0. Per Scott D, test with 1.65.0 first; this doc reflects that test. If this PR is accepted, the recommendation will be 1.66.0+***
```

Comment:
Clarification - by "this PR," I mean Scott's PR: https://github.com/ripple/rippled/pull/2348